src/routes/LoginRoute/index.js

```diff
@@ -0,0 +1,322 @@
+      validteCredentials({ email: true, password: false });
+    else if (
+      passwordInput.current.value === "" &&
+      emailInput.current.value !== ""
+    )
+      validteCredentials({ email: false, password: true });
+    else if (
+      passwordInput.current.value === "" &&
+      emailInput.current.value === ""
+    )
+      validteCredentials({ email: true, password: true });
+    else {
+      try {
+        const loginResponse = await loginUser({
+          email: emailInput.current.value,
+          password: passwordInput.current.value,
+        });
+        setLogin(true);
+        navigate(location.state?.from?.pathname ?? "/", { replace: true });
+      } catch (e) {
+        // validteCredentials({ email: null, password: null });
+        emailInput.current.value = "";
+        passwordInput.current.value = "";
+        setTestCredentials({
+          tesetEmail: "adarshbalika@gmail.com",
+          testPassword: "adarshBalika1234",
+        });
+        if (e.response.status === 401) {
+          toast.error("Invalid email or password. Please try again.");
+        } else if (e.response.status === 404) {
+          toast.error("No user found with this email. Please try again.");
+        } else {
+          toast.error("Unexpected error. Please try again in some time.");
+        }
+      }
+    }
+  };
+
+  return (
+    <div className="auth-form">
+      <h1>
+        Spark Wind <GiWindSlap className="hero-icon" />
+      </h1>
+      <form onSubmit={submitLoginForm}>
+        <center>
+          <h2>Login</h2>
+        </center>
+        <div>
+          <label>Email address</label>
+          <input
+            type="email"
+            placeholder="email"
+            ref={emailInput}
+            value={testCredentials && testCredentials.tesetEmail}
+            onChange={() =>
+              validteCredentials((prev) => ({
+                ...prev,
+                email: false,
+              }))
+            }
+          />
+          {validatedCredentials.email && (
+            <p className="style-error">Please Provide Email</p>
+          )}
+        </div>
+        <div>
+          <label>Password</label>
+          <input
+            type="password"
+            placeholder="********"
+            ref={passwordInput}
+            value={testCredentials && testCredentials.testPassword}
+            onChange={() =>
+              validteCredentials((prev) => ({
+                ...prev,
+                password: false,
+              }))
+            }
+          />
+          {validatedCredentials.password && (
+            <p className="style-error">Please Provide password</p>
+          )}
+        </div>
+        <div className="flex-H-space-bw">
+          <div className="flex-H-center-V">
+            <input type="checkbox" className="checkbox" />
+            <p>Remember Me</p>
+          </div>
+          <p>Forgot Your Password</p>
+        </div>
```

Comment:
A suggestion : if functionality of remember me and forget password wont be made.. It can be removed from UI